### ISSUE TITLE
Maven: Not Ansi highlighting anymore

### ISSUE THREAD

**Issue by melloware** [quarkusio/quarkus#32624]
### Describe the bug

Running the same project in Quarkus 2.16.6 and 3.0.0.Final I can see Ansi highlighting has been removed.

**2.16.6**
![image](https://user-images.githubusercontent.com/4399574/231811237-cb39eb6c-d00c-4690-9a88-469183adc72f.png)

**3.0.0**
![image](https://user-images.githubusercontent.com/4399574/231811404-3ffe5ecd-4bba-43bb-910d-bad5347219f1.png)

### Expected behavior

ANSI highlighting in `quarkus:dev` mode.

### Actual behavior

Not highlighiting

### Output of `uname -a` or `ver`

Microsoft Windows [Version 10.0.22621.1555]

### Output of `java -version`

OpenJDK Runtime Environment Temurin-17.0.6+10 (build 17.0.6+10)

### Quarkus version or git rev

3.0.0.Final

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

Apache Maven 3.9.1 (2e178502fcdbffc201671fb2537d0cb4b4cc58f8)

**Comment by melloware** [comment]
Looks like it was this commit: https://github.com/quarkusio/quarkus/pull/32137/files#diff-dbfecb8ead57f5ad197c876b824b224ea62a3711ba4a3d2c0bb07da027aa11d1

**Comment by gsmet** [comment]
Yes, that's what I had in mind. Could you try to debug what's going on in `saveTerminalState`?

**Comment by melloware** [comment]
yep i will try and take a look

**Comment by gsmet** [comment]
ah wait, I think I did something stupid

**Comment by gsmet** [comment]
Let me prepare a PR for you to test.

**Comment by gsmet** [comment]
@melloware could you try https://github.com/quarkusio/quarkus/pull/32626 and see how it goes?

**Comment by melloware** [comment]
@gsmet more info in this ticket: https://github.com/quarkusio/quarkus/issues/37373

Apparently `%dev.quarkus.log.console.color=true` makes it work on Windows?

**Comment by geoand** [comment]
Is this still an issue?

**Comment by melloware** [comment]
Yep this still is but i think we can close it since I am the only person left using Windows :)  And it has a workaround of `%dev.quarkus.log.console.color=true`

**Comment by geoand** [comment]
🙏
